AggregateMetrics/AggregateMetrics/AzureWebApp/SumUpGauge.cs

```diff
@@ -0,0 +1,51 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.AggregateMetrics.AzureWebApp
+{
+    using System.Collections.Generic;
+    using Microsoft.ApplicationInsights.DataContracts;
+    using Microsoft.ApplicationInsights.Extensibility.AggregateMetrics.Two;
+
+    /// <summary>
+    /// Gauge that sums up the values of different gauges.
+    /// </summary>
+    public class SumUpGauge : ICounterValue
+    {
+        private string name;
+
+        private readonly List<FlexiblePerformanceCounterGauge> gaugesToSum;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="SumUpGauge"/> class.
+        /// </summary>
+        /// <param name="name"> Name of the SumUpGauge.</param>
+        /// <param name="gauges"> Gauges to sum.</param>
+        public SumUpGauge(string name, params FlexiblePerformanceCounterGauge[] gauges)
```

Comment:
thinking about it - you don't need to strongly type `FlexiblePerformanceCounterGauge` - you only need an `ICounterValue`